src/OAuth2.php

```diff
@@ -766,7 +770,7 @@ public function setRedirectUri($uri)
      */
     public function getScope()
     {
-        if (is_null($this->scope)) {
+        if (is_null($this->scope) || is_string($this->scope)) {
```

Comment:
The typehint is wrong, the `$scope` property should always be an array. What prompted this change?

Reply:
fixed (and fixed the property type)